workloads/network-perf/run_hostnetwork_network_test_fromgit.sh

```diff
@@ -62,6 +63,10 @@ sleep 30
 
 uperf_state=1
 for i in {1..240}; do
+  if [ "$(oc get benchmarks.ripsaw.cloudbulldozer.io -n my-ripsaw -o json | jq -r .items[].status.state)" == "Error" ]; then
```

Comment:
jq might or might not be installed, maybe leverage the jsonpath to capture the state: https://kubernetes.io/docs/reference/kubectl/jsonpath/. Thoughts?

Reply:
yup, jsonpath++
updated